site/components/Provider/Provider.tsx

```diff
@@ -38,6 +39,7 @@ const connectors = connectorsForWallets([
       omniWallet({ chains }),
       imTokenWallet({ chains }),
       ledgerWallet({ chains }),
+      zerionWallet({ chains }),
```

Comment:
Reverting this last addition to the docs site should wrap that part up; only because the extension is not yet available.

Reply:
Ok!